docs/getting-started/detections-req.asciidoc

```diff
@@ -52,49 +54,72 @@ NOTE: For instructions about using Machine Learning jobs and rules, refer to <<m
 a|The `manage`, `write`,`read`, and `view_index_metadata` index privileges for the following system indices:
 
+* `.alerts-security.alerts-*`
 * `.siem-signals-*`
 * `.lists-*`
 * `.items-*`
-|{kib} space `All` privileges for the `Security` feature (see
+
+*NOTE*: If you upgrading to {stack} 8.0.0, it is recommended that users have privileges to the `.alerts-security.alerts-<kib-space>` AND `.siem-signals-*` index. If you are newly installing the {stack}, then users do not need privileges to the `.siem-signals-*` index.
+
+
+|{kib} space `All` privileges for the `Security` feature (visit
+{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges])
+
+
+| Preview rules
+|N/A
+a| The `read` privilege to the `.preview.alerts-security.alerts-<kib-space>` index
+|{kib} space `All` privileges for the `Security` feature (visit
 {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges])
 
 |Manage rules
 | N/A
 a|The `manage`, `write`,`read`, and `view_index_metadata` index privileges for the for the following system indices where `<kib-space>` is the {kib} space name:
 
+* `.alerts-security.alerts-*`
 * `.siem-signals-<kib-space>`
 * `.lists-<kib-space>`
 * `.items-<kib-space>`
 
-| {kib} space `All` privileges for the `Security` feature (see
+*NOTE*: If you upgrading to {stack} 8.0.0, it is recommended that users have privileges to the `.alerts-security.alerts-<kib-space>` AND `.siem-signals-*` index. If you are newly installing the {stack}, then users do not need privileges to the `.siem-signals-*` index.
+
+| {kib} space `All` privileges for the `Security` feature (visit
 {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges])
 
-*Note*: To manage rule connectors, your role will also need `All` privileges for the `Action and Connectors` feature (*Management* -> *Action and Connectors*).
+*NOTE*: To manage rule connectors, your role will also need `All` privileges for the `Action and Connectors` feature (*Management* -> *Action and Connectors*).
 
 |Manage alerts
 
-**Note**: Allows you to manage alerts, but not modify rules.
+**NOTE**: Allows you to manage alerts, but not modify rules.
 |N/A
 a|The `maintenance`, `write`,`read`, and `view_index_metadata` index privileges for the following system indices where `<kib-space>` is the {kib} space name:
 
+* `.alerts-security.alerts-<kib-space>`
+* `.internal.alerts-security.alerts-<kib-space>`
 * `.siem-signals-<kib-space>`
 * `.lists-<kib-space>`
 * `.items-<kib-space>`
-|{kib} space `All` privileges for the `Security` feature (see
+
+*NOTE*: If you upgrading to {stack} 8.0.0, it is recommended that users have privileges to the `.alerts-security.alerts-<kib-space>` AND `.siem-signals-*` index. If you are newly installing the {stack}, then users do not need privileges to the `.siem-signals-*` index.
+|{kib} space `All` privileges for the `Security` feature (visit
```

Comment:
```suggestion
|{kib} space `Read` privileges for the `Security` feature (visit
```
To match with the `Note` above about `Allows you to manage alerts, but not modify rules.` this should be `Read` privileges.